src/sentry/templates/sentry/emails/_suspect_commits.html

```diff
@@ -0,0 +1,23 @@
+{% load sentry_avatars %}
```

Comment:
could you put a screenshot of what this looks like in an email? I think we have test endpoints where we can see what the renders look like. https://github.com/getsentry/sentry/blob/56ef263316fa056a05094f8467b0297e246100bb/src/sentry/web/debug_urls.py#L77